cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeries.java

```diff
@@ -218,7 +228,16 @@ private List<Column> getColumnDescriptor() {
                 columns.add(new TimeSeries.Column(fieldName, fieldIndex + 1, f.getType()));
             }
         }
-
+        if (includeDataEntryDate) {
```

Comment:
This could also be accomplished better with a subclass

Reply:
Moved into subclass